react/components/BuyButton/Wrapper.js

```diff
@@ -129,8 +129,9 @@ const BuyButtonWrapper = ({
 
   const version =
     checkoutVersion &&
-    checkoutVersion.installedApp &&
-    checkoutVersion.installedApp.version
+    checkoutVersion.installedAppPublic &&
```

Comment:
Shouldn't you also enable `ssr` in the `withCheckoutVersion` hook?

Reply:
I will leave ssr false for now